pkg/query/options.go

```diff
@@ -45,11 +53,20 @@ type QueryBackendOptions struct {
 	// e.g. "/prediction/speccost"
 	PredictSpecCostPath string
 
+	// A boolean value  to automatically set parameters according to OpenCost specification.
+	OpenCost bool
+
 	restConfig *rest.Config
 	pfQuerier  *PortForwardQuerier
 }
 
 func (o *QueryBackendOptions) Complete(restConfig *rest.Config) error {
+	if o.OpenCost {
+		o.ServiceName = OpenCostServiceName
+		o.KubecostNamespace = OpenCostServiceNamespace
+		o.ServicePort = OpenCostServicePort
+		o.AllocationPath = OpenCostAllocationPath
+	}
```

Comment:
Nicely done, I appreciate that the opencost defaults first, and then additional flags can override field-by-field.